sprints/dashboard/libs/jira.py

```diff
@@ -107,6 +112,25 @@ def report(self, from_: str, to: str) -> Report:
         report = Report(self._options, self._session, r_json)
         return report
 
+    def worklog_list(self, worklogs: List[int]) -> List[Worklog]:
+        """
+        Retrieves list of worklogs with IDs provided in the request's body.
+
+        Limited to 1000 worklogs. Source:
+        https://developer.atlassian.com/cloud/jira/platform/rest/v3/?utm_source=%2Fcloud%2Fjira%2Fplatform%2Frest%2F&utm_medium=302#api-rest-api-3-worklog-list-post
+        """
+        aggregated_worklogs: List[Dict] = []
+        for chunk in chunks(worklogs, 1000):
+            r_json = self._session.post(
+                url=self._get_url('worklog/list', self.API_V2),
+                data=json.dumps({'ids': chunk}),
+            )
```

Comment:
Can these queries be crashing Jira?

Reply:
It's public Jira API, so we can assume it's stable. I tested this locally for all worklogs since 2015 (which is also included in the testing instructions in point 5) and it worked fine.